sentry-samples/sentry-samples-android/src/main/java/io/sentry/sample/MainActivity.java

```diff
@@ -74,18 +87,39 @@ protected void onCreate(Bundle savedInstanceState) {
 
     binding.anr.setOnClickListener(
         view -> {
+          SentryTransaction anrTransaction = Sentry.startTransaction("MainActivity.ANR");
+          Span anrSpan = anrTransaction.startChild();
+          anrSpan.setOperation("button");
+          anrSpan.setDescription("ANR demo button");
           // Try cause ANR by blocking for 2.5 seconds.
           // By default the SDK sends an event if blocked by at least 4 seconds.
           // The time was configurable (see manifest) to 1 second for demo purposes.
           // NOTE: By default it doesn't raise if the debugger is attached. That can also be
           // configured.
           try {
             Thread.sleep(2500);
+            anrSpan.setStatus(SpanStatus.OK);
           } catch (InterruptedException e) {
+            anrSpan.setStatus(SpanStatus.ABORTED);
             Thread.currentThread().interrupt();
           }
+          finally {
+            anrSpan.finish();
+            anrTransaction.finish();
+            Sentry.captureTransaction(anrTransaction, null);
+          }
         });
+    innerSpan.setStatus(SpanStatus.OK);
+    innerSpan.finish();
 
+    innerSpan = activityTransaction.startChild();
+    innerSpan.setOperation("view");
+    innerSpan.setDescription("setContentView");
     setContentView(binding.getRoot());
+    innerSpan.setStatus(SpanStatus.OK);
+    innerSpan.finish();
+
+    activityTransaction.finish();
+    Sentry.captureTransaction(activityTransaction, null);
```

Comment:
we should not require a `, null`, so an overload will be handy.

Reply:
I'll add it.